docs/alerts.md

```diff
@@ -0,0 +1,65 @@
+# Alerts
+
+<!-- TOC depthTo:2 -->
+
+- Elasticsearch cluster
+    - [ElasticsearchClusterNotHealthyRed](#ElasticsearchClusterNotHealthyRed)
+    - [ElasticsearchClusterNotHealthyYellow](#ElasticsearchClusterNotHealthyYellow)
+    - [ElasticsearchWriteRequestsRejectionJumps](#ElasticsearchWriteRequestsRejectionJumps)
+    - [ElasticsearchNodeDiskWatermarkDiskLowReached](#ElasticsearchNodeDiskWatermarkDiskLowReached)
+    - [ElasticsearchNodeDiskWatermarkDiskHighReached](#ElasticsearchNodeDiskWatermarkDiskHighReached)
+    - [ElasticsearchNodeDiskWatermarkFloodReached](#ElasticsearchNodeDiskWatermarkFloodReached)
+    - [ElasticsearchJVMHeapUseHigh](#ElasticsearchJVMHeapUseHigh)
+    - [AggregatedLoggingSystemCPUHigh](#AggregatedLoggingSystemCPUHigh)
+    - [ElasticsearchProcessCPUHigh](#ElasticsearchProcessCPUHigh)
+    - [ElasticsearchDiskSpaceRunningLow](#ElasticsearchDiskSpaceRunningLow)
+    - [ElasticsearchHighFileDescriptorUsage](#ElasticsearchHighFileDescriptorUsage)
+
+<!-- /TOC -->
+
+## ElasticsearchClusterNotHealthyRed
+
+Cluster health status is RED. At least one primary shard and its replicas are not allocated to a node.
```

Comment:
Can we rephrase this slightly and also add some preliminary "things to check for" (even if its sparse initially)

```
## Elasticsearch Cluster Health is Red

At least one primary shard and its replicas are not yet allocated to a node.

### Troubleshooting
```

Reply:
This could apply to the other sections as well

Reply:
Updated all section titles with the troubleshooting subsection. Please review the change. Thanks.